pages/blog/hacktoberfest-2022.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "Is Hacktoberfest Good For Maintainers?"
```

Comment:
Is this the correct title for this PR? 🤔 Because in the blog post, you are summarising Hacktoberfest 2022 in AsyncAPI.

Reply:
yup, i think it is title of old existing article

Reply:
I copied from the old one with updating in mind and I forgot 😄